torch/_tensor_str.py

```diff
@@ -142,6 +144,9 @@ def format(self, value):
                     ret += '.'
             else:
                 ret = ('{{:.{}f}}').format(PRINT_OPTS.precision).format(value)
+        elif self.complex_dtype:
+            p = PRINT_OPTS.precision
+            ret = '({{:.{}f}} {{}} {{:.{}f}}j)'.format(p, p).format(value.real, '+-'[value.imag < 0], abs(value.imag))
```

Comment:
@anjali411 

Does `value.real` instead of `value.real()` work now? I had some problems adding this without breaking some tests. `value.real` is more compatible with numpy.

Reply:
yeah value.real and value.imag work well. Just tried, value.real() .imag(), but got an error they probably don't have the functions anymore?

Reply:
Interesting, good to know that got fixed.